stl/inc/ostream

```diff
@@ -109,7 +109,11 @@ public:
         _STL_DISABLE_DEPRECATED_WARNING
         __CLR_OR_THIS_CALL ~sentry() noexcept {
 #if _HAS_EXCEPTIONS
+#if _HAS_DEPRECATED_UNCAUGHT_EXCEPTION
             if (!_STD uncaught_exception()) { // TRANSITION, ArchivedOS-12000909
+#else // ^^^ _HAS_DEPRECATED_UNCAUGHT_EXCEPTION / !_HAS_DEPRECATED_UNCAUGHT_EXCEPTION vvv
+            if (!_STD uncaught_exceptions()) {

```

Comment:
Style issue: We generally avoid negating integers. I believe that this should be phrased as `if (_STD uncaught_exceptions() == 0)`.

Reply:
I switched both to `_STD uncaught_exception{s}() == 0` so that they are consitent

Reply:
The issue is that the return types aren't consistent - `uncaught_exception()` returns `bool` while `uncaught_exceptions()` returns `int`. Stylistically, we never compare `bool` to `0`, so I'm going to restore that line.